pkg/modulewriter/modulewriter_test.go

```diff
@@ -500,6 +491,7 @@ func (s *MySuite) TestWriteMain(c *C) {
 		ID: "test_module",
 		Settings: map[string]interface{}{
 			"testSetting": "testValue",
+			"passthrough": `(("${vars.deployment_name}-allow\"))`,
```

Comment:
I want to certain what you're testing here. You are testing a Go literal string that includes `\` and `$` ensuring that `hclwrite` (and our layers above it), do not escape them?